Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:current-alpine AS base
```

Comment:
Would be nice to name this stages 🥰 
See [here](https://docs.docker.com/develop/develop-images/multistage-build/#name-your-build-stages)

Reply:
From the [Docker document](https://docs.docker.com/develop/develop-images/multistage-build/#name-your-build-stages). "By default, the stages are not named. However, you can name your stages, by adding an AS <NAME>" So I named it as `BASE`. Anyway, it's really optional.